File: examples/regex_match_store/regex_match_store.php
<?php
$value = "This is target 1.
Name is apple.
Target 2 is here. 
Name is orange.";
$matches = [];
preg_match_all('/target\s([\d+]).*?Name\s+is\s+([^\.]+)\./is', $value, $matches);
if(isset($matches[1])) {
    $i = 0;
    foreach($matches[1] as $match) {
        print($match."=".$matches[2][$i]."\n");
        $i++;
    }
}

File: examples/regex_match_store/regex_match_store.rb
value = "This is target 1.
Name is apple.
Target 2 is here. 
Name is orange.";
matches = value.scan(/target\s([\d+]).*?Name\s+is\s+([^\.]+)\./im)
for match in matches do
  puts(match[0]+"="+match[1])
end
